client/a8c-for-agencies/sections/marketplace/controller.tsx

```diff
@@ -18,17 +19,19 @@ export const marketplaceContext: Callback = () => {
 
 export const marketplaceProductsContext: Callback = ( context, next ) => {
 	const { site_id, product_slug, purchase_type } = context.query;
+	const isAutomatedReferrals = isEnabled( 'a4a-automated-referrals' );
 	const productBrand = context.params.brand;
 
 	context.secondary = <MarketplaceSidebar path={ context.path } />;
-	const purchaseType = purchase_type === 'referral' ? 'referral' : 'regular';
+	const purchaseType =
+		!! isAutomatedReferrals && purchase_type === 'referral' ? 'referral' : 'regular';
```

Comment:
[nitpick] isEnabled( 'a4a-automated-referrals' ) gives a boolean value.

```suggestion
	const purchaseType =
		isAutomatedReferrals && purchase_type === 'referral' ? 'referral' : 'regular';
```